File: P1-Linguagem_de_Programação_I/Desafios dos Roteiros - C++/desafio-roteiro06.cpp/versao-modular/ex06.cpp
#include <iostream>
using namespace std;

float aplicar_desconto(float valor, float desconto) {
    return valor - (valor * desconto / 100);
}

int main() {

    float valor, desconto;

    cout << "Digite o valor do produto: ";
    cin >> valor;

    cout << "Digite o desconto (%): ";
    cin >> desconto;

    cout << "Valor final: " << aplicar_desconto(valor, desconto);

    return 0;
}

File: P1-Linguagem_de_Programação_I/Desafios dos Roteiros - Python/desafio-roteiro06/versao-modular/ex06.py
def aplicar_desconto(valor, desconto):
    return valor - (valor * desconto / 100)

valor = float(input("Digite o valor do produto: "))
desconto = float(input("Digite o percentual de desconto: "))

resultado = aplicar_desconto(valor, desconto)

print("Valor final:", resultado)